modules/unrulyBidAdapter.js

```diff
@@ -4,9 +4,12 @@ import { registerBidder } from '../src/adapters/bidderFactory'
 import { VIDEO } from '../src/mediaTypes'
 
 function configureUniversalTag (exchangeRenderer) {
+  if (!exchangeRenderer.config) throw new Error('UnrulyBidAdapter: Missing renderer config.')
+  if (!exchangeRenderer.config.siteId) throw new Error('UnrulyBidAdapter: Missing renderer siteId.')
```

Comment:
@paprikka 

Are these checks now irrelevant?

Due to the filter, it seems a bid would never come into this function unless it already has the config and siteID.

Obviously this is not a show-stopper, but if it is irrelevant, please remove it.

If you do not get to it before the release scheduled for tomorrow, I will make sure to still merge it and we can update in a later PR.

Reply:
Ah, yes—these are not required any more. 
I'm afraid I don't time to deal with this today—will submit another PR later this week if that's ok?

Reply:
Yes, please do not forget to update it when you have a chance!